File: app/src/main/java/com/u1fukui/bbs/model/Comment.java
package com.u1fukui.bbs.model;

import java.io.Serializable;
import java.util.Date;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class Comment implements Serializable {

    private static final long serialVersionUID = -2300031079350686170L;

    public long id;

    public long threadId;

    public int displayNumber;

    public String description;

    public User author;

    public int likeCount;

    public boolean isLiked;

    public Date createdAt;
}

File: app/src/main/java/com/u1fukui/bbs/model/Comment.kt
package com.u1fukui.bbs.model

import se.ansman.kotshi.JsonSerializable
import java.io.Serializable
import java.util.*

@JsonSerializable
data class Comment(
        val id: Long,
        val threadId: Long,
        val displayNumber: Int,
        val description: String,
        val author: User,
        val likeCount: Int,
        val isLiked: Boolean,
        val createdAt: Date
) : Serializable {

    companion object {

        private const val serialVersionUID = -2300031079350686170L
    }
}
